sdk/core/azure-core/azure/core/pipeline/transport/_requests_asyncio.py

```diff
@@ -30,8 +30,6 @@
 from typing import Any, Union, Optional, AsyncIterator as AsyncIteratorType
 import urllib3 # type: ignore
 
-import requests
```

Comment:
Given this file is about requests, I would keep it up by principle. It might mean we need the 3.7 perf system for the entire file if we want benefits here.